src/SIM/SIMoutput.h

```diff
@@ -289,6 +289,9 @@ class SIMoutput : public SIMinput
   //! \brief Returns the reference norm to base mesh adaptation upon.
   virtual double getReferenceNorm(const Vectors& gNorm, size_t adaptor) const;
 
+  //! \brief Returns the reference norm for effectivity index.
+  virtual double getEfficiencyNorm(const Vectors& gNorm, size_t adaptor) const;
+
```

Comment:
Hmm, think instead maybe I prefer a
`virtual double getEffectivityIndex(const Vectors& gNorm, size_t idx, size_t inorm) const`
with the default implementation:
```
double SIMoutput::getEffectivityIndex (const Vectors& gNorm, size_t idx, size_t inorm) const
{
  return gNorm[idx](inorm)/gNorm.front()(4);
}
```

Reply:
you mean beyond column 80? Then insert newline where most appropriate (I did not consider the formatting here)

Reply:
yes that was for the SIMbase comment. i couldn't comment directly on it for some reason. i will revert that so it looks as it did originally then.